src/cache-restore.ts

```diff
@@ -36,12 +36,12 @@ export const restoreCache = async (
     );
   }
 
-  const primaryKey = `node-cache-${platform}-${packageManager}-${fileHash}`;
+  const primaryKey = `node-cache-${platform}-${packageManager}-v2-${fileHash}`;
```

Comment:
Could you please change the key. We do not need to change key because if customers get different cache paths it will resave cache so we can keep cache key as it is.

Reply:
@MaksimZhukov  what's your opinion?

Reply:
I think we can get rid of this, as suggested by Dmitry